### ISSUE TITLE
FR: Editing ACL tags for an object should have a text / autocomplete field

### ISSUE THREAD

**Issue by locriani** [tailscale/tailscale#12855]
### What are you trying to do?

In the edit ACL tag UI on the web, it is currently very painful to select a specific ACL tag if you have a large list of tags.

![IMG_1710](https://github.com/user-attachments/assets/ebdd2cd5-761c-4fa3-a06e-6d95fba6ff69)

### How should we solve this?

An autocomplete text field would be very useful / helpful here.

### What is the impact of not solving this?

Manually scrolling through a large list of tags is very painful.

**Comment by bradfitz** [comment]
Corp bug (Nov 2023) is https://github.com/tailscale/corp/issues/15567

**Comment by Radiergummi** [comment]
Yes please. I've just started to setup Tailscale for my org, and will have to set up about 80 machines. Each of them has a handful of tags. I'm dreading the hours I have to spend clicking (and scrolling!!) that darn dropdown. This should be a *way* faster operation.
